feat: configure LDP server with certificate + some cleanup (#72)
salesforcecli/plugin-lightning-dev: messages/lightning.preview.app.md

Downloading

# certificate.installation.notice

> A self-signed certificate needs to be installed on your device before you can use your device for previewing. This is a one-time setup only which you can skip if you have installed the certificate already.

Reviewer: Could something like this be a little more concise and still accurate?

```suggestion
To use local preview on your device, you have to install a self-signed certificate on it. If you previously set up a certificate for your device, you can skip this step.
```

# certificate.installation.skip.message

Do you want to skip this step

# certificate.installation.description

> Please install the self-signed certificate on your device before proceeding. The certificate file is located at

Reviewer: I think we usually exclude terms like "please" -- how does this look?

```suggestion 
Before proceeding, install the self-signed certificate on your device. The certificate file is located at
```

> 

Author: @moorejacqueline As I mentioned in a [comment on your PR](https://github.com/salesforcecli/plugin-lightning-dev/pull/64#discussion_r1645232377), I updated the strings to address a UX feedback that we had received. The feedback was to first show a message to the user indicating the need for installing a certificate (one time action) and ask if they've done this already and want to skip or not. If they choose to not skip then show instructions on how to install.

Please review these strings and let me know if anything needs to change.

Reviewer: Thanks for tagging me! I'll add a few small suggestions to the new text -- I think I can't do it in this thread.
